src/InputForm.tsx

```diff
@@ -0,0 +1,60 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react'
+import { Button, Cell, Grid, HFlow, TextField, VFlow } from 'bold-ui'
+import React, { useState } from 'react'
+
+function InputForm() {
+  const [formState, setFormState] = useState({
+    input: '',
+  })
```

Comment:
```suggestion
  const [formState, setFormState] = useState('')
```
Como só tem a string, por enquanto não precisa ser um objeto né